README.md

```diff
@@ -4,13 +4,26 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. The smaller order variables are ignored.
+  2. Coefficents are ignored.
+  3. Asymptotic analysis considers worst, average, and best times which can be misleading by overestimating or underestimating the time of the program.
```

Comment:
How can this overestimate or underestimate? Of course the best case doesn't tell you anything about the worst case, but that's by definition.